### ISSUE TITLE
upload missing code

### ISSUE THREAD

**Issue by LilySnow** [DeepRank/deeprank#194]

**Comment by LilySnow** [comment]
@CunliangGeng Do you have code for generating pssm for 3DeepFace?

**Comment by CunliangGeng** [comment]
Yes, I should have it on alcazar. But I don't think it's really necessary to provide these scripts considering we already have the PSSMGen package.
